Dockerfile

```diff
@@ -3,11 +3,11 @@ FROM ubuntu:16.04
 ENV QT_VERSION v5.9.1
 ENV QT_CREATOR_VERSION v4.3.1
 
-RUN apt-get -y upgrade
 RUN apt-get -y update
+RUN apt-get -y upgrade
 
 # Build prerequisites
-RUN apt-get -y build-dep qt5-default
+RUN apt-get -y install qt5-default make g++ python
```

Comment:
I may have misunderstood the build-dep command. Checking if qt5-default is needed and will remove if not.

Reply:
Changing the apt-get command from *build-dep* to *install* will install the package rather than install the dependencies needed to build the package. This is not what we want, as we are buiding Qt from source.

I prefer to use the *build-essential* package instead of installing make, gcc etc. individually.